setup.py

```diff
@@ -28,8 +28,9 @@
     long_description = f.read()
 
 package_name = "dbt-glue"
-package_version = "0.2.14"
-dbt_version = "1.2.0"
+package_version = "0.2.15"
+dbt_min_version = "1.2.0"
+dbt_max_version = "1.3.2"
```

Comment:
why that? do you want to maintain cross compatibility? I would rather suggest to kept it easy and just add `dbt_version = "1.3.0"`

Reply:
I did that as currently it is compatible with both versions. So if someone does not yet want to update they can have the lower version of dbt-core. At the point it become incompatible then we should update. 
The other option which may be more contentious would be to bump the version to 1.3.0 like the main dbt packages do to show that this branch follows the 1.3 version of dbt-core?